# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1295549263 28800
#      Thu Jan 20 10:47:43 2011 -0800
# Node ID 21dec063e89fcf712c3ceb7c8af7d872f9db8311
# Parent  6006ca07410d3a92a3b47f2984a1513bf27f7361
More updates to top-level README documentation files ahead of 3.4 release.

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,15 @@
+2010-01-20  Rik  <octave@nomad.inbox5.com>
+
+	* README.Cygwin, README.MacOS, README.Windows, README.ftp: Update
+	documentation files for 3.4 release.
+
+	* README.MinGW: Create placeholder for new README file.
+
 2011-01-20  John W. Eaton  <jwe@octave.org>
 
 	* bootstrap: Update from gnulib sources.
 
 2011-01-20  John W. Eaton  <jwe@octave.org>
 
 	* configure.ac, NEWS: Update for backend -> graphics_toolkit change.
 
diff --git a/README.Cygwin b/README.Cygwin
--- a/README.Cygwin
+++ b/README.Cygwin
@@ -20,34 +20,32 @@ configure --enable-shared \
           LDFLAGS=-no-undefined 
 
 "--enable-float-truncate" is needed for the following bug:
 http://thread.gmane.org/gmane.comp.gnu.octave.bugs/12361/focus=12404
 Without it, one of the quadgk test will fail as
 "a=a" could be false due to truncation problems with 
 complex numbers. 
 
- 
 "lt_cv_deplibs_check_method=pass_all"  is needed to bypass
 incorrect libtool detection of system capabilities and
 to allow shared libs building.
 
+
 Octave-3.2.4 was built using:
 
 configure --enable-shared \
           --without-fltk \
           --without-framework-opengl \
           CC=gcc-4 F77=gfortran-4 CXX=g++-4 CPP=cpp-4
           CFLAGS="-Dtimezone=_timezone"
 
 
-
- 
 Current Cygwin package maintainer for Octave:
 
   Marco Atzeri
   http://matzeri.altervista.org
 
 Marco Atzeri
 marco_atzeri@yahoo.it
 Italy
 
-Mon Jan  3 18:53:41 WEST 2011
+Last updated: Mon Jan  3 18:53:41 WEST 2011
diff --git a/README.MacOS b/README.MacOS
--- a/README.MacOS
+++ b/README.MacOS
@@ -135,9 +135,9 @@ 2.5 Building without a package manager
 
 
 John W. Eaton
 jwe@octave.org
 
 Ben Abbott
 bpabbott@mac.com
 
-Sat, 28 Oct 2010 16:06:00 EDT
+Last updated: Sat, 28 Oct 2010 16:06:00 EDT
diff --git a/README.MinGW b/README.MinGW
new file mode 100644
--- /dev/null
+++ b/README.MinGW
@@ -0,0 +1,1 @@
+Placeholder until file written by Tatsuro and reviewed.
diff --git a/README.Windows b/README.Windows
--- a/README.Windows
+++ b/README.Windows
@@ -1,11 +1,11 @@
 See the file README.Cygwin for information about installing the binary
 package of Octave for Cygwin.
 
-See the file README.MSVC for instructions for compiling Octave with
-the MSVC compiler.
+See the file README.MinGW for information about installing the binary
+package of Octave for MinGW.
 
 
 John W. Eaton
 jwe@octave.org
 
-Last updated: Wed, 31 Oct 2007 16:34:12 EDT
+Last updated: Thu Jan 20 10:35:23 PST 2011
diff --git a/README.ftp b/README.ftp
--- a/README.ftp
+++ b/README.ftp
@@ -1,23 +1,25 @@
 This directory contains the source for Octave, a high-level interactive
-language for solving numerical problems.  See the files README.octave
-and Announce for more general information, and the file NEWS for a
-list of recent changes.
+language for solving numerical problems.  See the file README for more general
+information, and the file NEWS for a list of recent changes.
 
 Binary distributions:
 --------------------
 
 Octave binaries are not distributed from this site.
 
 Packaged versions of Octave for various GNU/Linux systems are available
-with the major GNU/Linux distributions (Debian, Red Hat, SuSE, etc.).
+with the major GNU/Linux distributions (Debian, Ubuntu, Red Hat, SUSE, etc.).
 
-Binary distributions of Octave for Mac OS X are available as part of
-the fink project: http://fink.sourceforge.net
+Binary distributions of Octave for Mac OS X are available from both Fink
+and MacPorts.
+
+  http://www.finkproject.com
+  http://www.macports.org/
 
 The file README.Windows provides instructions for installing Octave on
 Windows systems.
 
 A note about .gz files:
 ----------------------
 
 Files with names ending in `.gz' have been compressed with `gzip'.
@@ -27,9 +29,10 @@ patents and tends to compress better any
 `compress'-compressed files too, so you can install it as "uncompress"
 and use it to handle both types of files.
 
 The gzip program is available in the directory /pub/gnu in shar, tar,
 or gzipped tar format (for those who already have a prior version of
 gzip and want faster data transmission).  It works on virtually every
 unix system, MSDOS, OS/2, and VMS.
 
-Last updated: Wed, 07 Apr 2010 15:15:04 EDT
+
+Last updated: Thu Jan 20 10:14:49 PST 2011
